### PR TITLE
Added default values

### PR DESCRIPTION
**Pull request by fredrik-jansson-se**

### CODE REVIEW

File: packages/sol6/src/yang/etsi-nfv-vnf.yang [hunk 1]
```diff
@@ -890,7 +890,7 @@ submodule etsi-nfv-vnf {
         leaf size-of-storage {
           type uint64;
           units "Gb";
-          mandatory true;
+          default 0;
```
Comment (mjethanandani) [inline packages/sol6/src/yang/etsi-nfv-vnf.yang]: Sorry. Just noticed this. Why is the default value 0, and not say 1. Are we expecting that NSD can do without any storage?

Comment (fredrik-jansson-se) [reply packages/sol6/src/yang/etsi-nfv-vnf.yang]: The reason is that 0 is a magic value in OpenStack, meaning pick whatever size is in the image.

Comment (fredrik-jansson-se) [reply packages/sol6/src/yang/etsi-nfv-vnf.yang]: And it works the same way (AFAIK) with VMware/OVA.

Comment (mjethanandani) [reply packages/sol6/src/yang/etsi-nfv-vnf.yang]: Ahh! I see. In that case I am fine with all the changes. Thanks.

**Pull request merged**